src/libmongoc/src/mongoc/mongoc-gridfs-bucket-file.c

```diff
@@ -367,12 +373,7 @@ _mongoc_gridfs_bucket_file_writev (mongoc_gridfs_bucket_file_t *file,
          space_available = file->chunk_size - file->in_buffer;
          to_write = _mongoc_min (bytes_available, space_available);
          memcpy (file->buffer + file->in_buffer,
-#ifndef _WIN32
-                 (char *)
-#endif
-                       iov[i]
-                          .iov_base +
-                    written_this_iov,
+                 IOV_MEMCPY_CAST (iov[i].iov_base) + written_this_iov,
```

Comment:
```suggestion
                 (char*) (iov[i].iov_base) + written_this_iov,
```

Consider unconditionally casting. `iov_base` is a `void *`. https://stackoverflow.com/a/3524270 notes arithmetic on `void *` is permitted through compiler extensions.